addon/components/validation-error-field.js

```diff
@@ -6,12 +6,18 @@ var ValidationErrorField = Ember.Component.extend({
     attributeBindings: ["array", "className", "model", "field", "validation", "submitted", "delayed", "index"],
     initialize: Ember.on("init", function() {
         var field = this.get("field");
-        Ember.Binding.from("model." + field).to("fieldName").connect(this);
-        Ember.Binding.from("model." + field + "IsPrimed").to("isPrimed").connect(this);
-
         var computedKeys = ["validation", "isPrimed", "submitted", "fieldName"];
+
         if(this.get("array")) {
-            computedKeys.push("array.@each." + field);
+            var fields = field.split(",");
+            fields.forEach(function(f) {
```

Comment:
About "only supporting list" when using `array` - do you mean we should split on "," above the conditional and throw an error of some kind (to show unsupported configuration) ?